src/pages/index.md

```diff
@@ -20,7 +20,7 @@ API Mesh for Adobe Developer App Builder is an orchestration layer that enables
 
 API Mesh allows developers to combine multiple data sources into a single, queryable GraphQL endpoint. GraphQL allows you to determine the information the response contains, unlike REST APIs, which have fixed response formats.
 
-By serving multiple sources through a single gateway and providing customized responses, API Mesh creates a better end-user experience for accessing your backend services.
+By serving multiple sources through a single gateway and providing customized responses, API Mesh creates a better end user experience for accessing your backend services.
```

Comment:
MS suggests that this should just be "user experience" https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/user-end-user

Reply:
I don't see the "user experience" suggestion on the page.

Reply:
@dshevtsov i mean that since Microsoft suggests using "user" instead of "end user", this quote in question would become:

```suggestion
By serving multiple sources through a single gateway and providing customized responses, API Mesh creates a better user experience for accessing your backend services.
```